src/juju/watchers.ts

```diff
@@ -128,18 +134,14 @@ export function processDeltas(
       Object.entries(applicationUnitCounts).forEach(
         ([applicationName, count]) => {
           if (!modelData.applications[applicationName]) {
-            // The following line is ignored because the type requires that
-            // the application has a number of pre-defined values.
-            // This is only here because sometimes the unit delta is parsed
-            // before the application delta arrives so it needs somewhere to
-            // store this information.
-            // This can be resolved by computing the unit-count in the
-            // selector instead of when the unit delta arrives.
-            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-            // @ts-ignore TSFixMe XXX
```

Comment:
Instead of ignore this, the application type has been updated to either be the full application object or just the unit count, as this ignore meant that if the application only had the unit count it could break the UI.